lib/Cake/View/Helper/FormHelper.php

```diff
@@ -664,6 +664,10 @@ protected function _secure($lock, $field = null, $value = null) {
 			$field = Hash::filter(explode('.', $field));
 		}
 
+		if (is_array($field)) {
+			$field = array_filter($field, 'strlen');
```

Comment:
When `$field` is string, wouldn't it be filtered twice, at line 664 and this line?